tests/isolation2/sql/test_dropped.sql

```diff
@@ -0,0 +1,22 @@
+-- Ensure diskquota does not save information about dropped table during restart cluster
```

Comment:
it is better to write that it invalidates at startup. also, if 'dropped table' is the name of the table from the test, write it with an underscore.

Reply:
> it is better to write that it invalidates at startup.

The invalidation occurs during startup, but the problem appeared when the cluster was restarted.



> if 'dropped table' is the name of the table from the test, write it with an underscore.

This means any dropped tables, not just the specific one from the test.

Reply:
but we are testing invalidate, so it's worth writing about it.

Reply:
> Ensure diskquota does not save information about the dropped table during restart cluster by invalidates it at startup.

Reply:
fixed